src/resources/views/crud/fields/upload.blade.php

```diff
@@ -16,7 +16,12 @@
         @if (isset($field['temporary']))
             <a target="_blank" href="{{ (asset(\Storage::disk($field['disk'])->temporaryUrl(Arr::get($field, 'prefix', '').$field['value'], Carbon\Carbon::now()->addMinutes($field['temporary'])))) }}">
         @else
-            <a target="_blank" href="{{ (asset(\Storage::disk($field['disk'])->url(Arr::get($field, 'prefix', '').$field['value']))) }}">
+            @if (Str::contains($field['value'], 'http'))
```

Comment:
What if the field value happens to have `http` but somewhere else, like in the file name? It's unlikely... but it's possible, right? So how about this, would this work too?

```suggestion
            @if (Str::startsWith($field['value'], 'http'))
```